api/src/main/java/io/minio/MinioClient.java

```diff
@@ -671,15 +671,7 @@ public MediaType contentType() {
 
```

Comment:
@balamurugana, if you focus on the function contentLength() itself, you can notice that the only change I've made is to return 0 instead of -1 when length == 0. Otherwise the behavior will be still the same.